modules/analysis/src/endpoints/test.rs

```diff
@@ -371,7 +379,7 @@ async fn test_retrieve_query_params_endpoint(ctx: &TrustifyContext) -> Result<()
     let uri = format!("/api/v2/analysis/root-component?q=sbom_id={}", sbom_id);
     let request: Request = TestRequest::get().uri(uri.clone().as_str()).to_request();
     let response: Value = app.call_and_read_body_json(request).await;
-    assert_eq!(&response["total"], 8);
+    assert_eq!(&response["total"], 9);
```

Comment:
why is the count changing ?

Reply:
To my understanding it's the "document node" the `SPDXRef-DOCUMENT` which is now returned too.